packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/SwatchPickerBestPractices.md

```diff
@@ -0,0 +1,29 @@
+<details>
+  <summary>
+    Best practices
+  </summary>
+
+### Do
+
+- Maximum recommended number of swatches per row is 8.
+- Use a grid layout when there are more than 4 swatches. The maximum recommended number of swatches in a grid is 64 - an 8 x 8 grid.
+- Tooltip should be shown for each color.
+
+### Don't
+
+- Avoid using rounded corners in a grid layout for the SwatchPicker, as it may cause the Hermann grid illusion.
```

Comment:
Can we add a link that references to an explanation of this Hermann Grid Illusion definition?